tests/test_group.py

```diff
@@ -2,11 +2,12 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import time
-from random import randrange
+import uuid
 
 import pytest
 
+from random import randrange
+
```

Comment:
There was no need to move this line down. Both `uuid` and `random` are standard library modules, so according to PEP8 these should be grouped together. See https://www.python.org/dev/peps/pep-0008/#imports for more details.